calcom/cal.com
# Only make web3 call if credential exists

## What does this PR do?



This PR adds the `hasweb3credential` prop to the event type page so it avoids making unnecessary calls.

Fixes #3207 



**Environment**: Staging(main branch) / Production

## Type of change




## How should this be tested?



- Go to an event type page without meta mask connected
- Check the console, token list should not be listed
- Go to the app store and connect meta mask
- Go to an event type page, token list should be logged
